src/work-stealing-queue.h

```diff
@@ -36,6 +36,13 @@ static inline ws_array_t *create_ws_array(size_t capacity, int32_t eltsz) JL_NOT
 
 typedef struct {
     _Atomic(int64_t) top;
+    // put `top` and `bottom` on different cache lines
+    // to avoid false sharing
+#ifdef _P64
+    void *_ignore[7];
```

Comment:
for aarch64 apple this needs to be 13

Reply:
We should be using `alignas` and the global define.